test/extended/networking/isolation.go

```diff
@@ -24,12 +28,20 @@ var _ = Describe("[networking] network isolation plugin", func() {
 		defer f1.Client.Pods(f1.Namespace.Name).Delete(podName, nil)
 		ip := launchWebserverPod(f1, podName, node1.Name)
 
-		By("Checking that the webserver is not accessible from a pod in a different namespace on the same node")
+		By("Trying to connect to the webserver from a pod in a different namespace on the same node")
```

Comment:
Consider separating the single node and multinode checks into separate tests so that isolation can be checked in a single node configuration and multinode isolation can be skipped if only a single node is available.  This is the pattern of the upstream version of the sanity checks that will eventually replace the checks in origin:

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/networking.go#L216